firmware/config/engines/mazda/mazda_miata_1_6.cpp

```diff
@@ -162,25 +162,11 @@ void miataNAcommonEngineSettings() {
 
 	// chartsize 200
 	engineConfiguration->engineChartSize = 200;
-
-	setTable(config->crankingCycleBaseFuel, 27.5);
-	config->crankingFuelCoef[0] = 2.8; // base cranking fuel adjustment coefficient
-	config->crankingFuelBins[0] = -20; // temperature in C
-	config->crankingFuelCoef[1] = 2.2;
-	config->crankingFuelBins[1] = -10;
-	config->crankingFuelCoef[2] = 1.8;
-	config->crankingFuelBins[2] = 5;
-	config->crankingFuelCoef[3] = 1.5;
-	config->crankingFuelBins[3] = 30;
-
-	config->crankingFuelCoef[4] = 1.0;
-	config->crankingFuelBins[4] = 35;
-	config->crankingFuelCoef[5] = 1.0;
-	config->crankingFuelBins[5] = 50;
-	config->crankingFuelCoef[6] = 1.0;
-	config->crankingFuelBins[6] = 65;
-	config->crankingFuelCoef[7] = 1.0;
-	config->crankingFuelBins[7] = 90;
+	constexpr float crankingBaseFuel = 27.5;
+	setTable(config->crankingCycleBaseFuel, crankingBaseFuel);
+	config->crankingCycleBaseFuel[0][0] = crankingBaseFuel * 2.8;
```

Comment:
We do not care to have legit canned tune

is the not-flat / not default crankingCycleBaseFuel useful for tests at all? if not, we shall simply remove all hard coded here

Reply:
Nope, it's not used in tests, I'll remove it now.